linebot/account_link_test.go

```diff
@@ -74,20 +89,22 @@ func TestIssueLinkToken(t *testing.T) {
 	}
 	for i, tc := range testCases {
 		currentTestIdx = i
-		res, err := client.IssueLinkToken(tc.UserID).Do()
-		if tc.Want.Error != nil {
-			if !reflect.DeepEqual(err, tc.Want.Error) {
-				t.Errorf("Error %d %v; want %v", i, err, tc.Want.Error)
-			}
-		} else {
-			if err != nil {
-				t.Error(err)
+		t.Run(strconv.Itoa(i)+"/"+tc.Label, func(t *testing.T) {
```

Comment:
Use `fmt.Sprintf` .

Reply:
IMO in this case `fmt.Sprintf` is too much.
It is not complex thing and faster than `fmt.Sprintf`.
https://gist.github.com/nasa9084/b2a54bec49bf4ede3b453e8b224ea3b8

If it is not complex string, we should avoid using `fmt.Sprintf`.

Reply:
Ok, I got it.
Thanks.